src/components/Pages/Guide/index.js

```diff
@@ -58,17 +59,15 @@ function Guide(props) {
     We want the GuideSection right before the first GuideSection that is past the window's position.
   **/
   function handleScroll() {
-    const thing = sectionLocations[0].offsetTop;
```

Comment:
@nick looks like this was a ruin of your testing, so I "put it out to pasture". 🌅

Reply:
you mean @Niicck

Reply:
Sorry @Nick! I'll look out for that in the future.
- But, you have a great Friday evening, ok!